k8s/namespaces/jenkins/patches/cftptl/cluster-00/jenkins.yaml

```diff
@@ -17,10 +17,10 @@ spec:
       resources:
         requests:
           cpu: "4"
-          memory: "20Gi"
+          memory: "24Gi"
         limits:
           cpu: "6"
-          memory: "20Gi"
+          memory: "24Gi"
```

Comment:
Didn’t we try 24 once and there wasn’t enough memory?

Reply:
Yes, it’s a bit of a gamble, but if those other workloads stay away from the node it should work as there are currently 28GB allocatable